contracts/src/gateway/GatewayGetterFacet.sol

```diff
@@ -20,6 +20,11 @@ contract GatewayGetterFacet {
     using EnumerableSet for EnumerableSet.AddressSet;
     using EnumerableSet for EnumerableSet.Bytes32Set;
 
+    /// @notice Returns code commit SHA where this contract is from.
+    function getCommitSha() external view returns (bytes32) {
```

Comment:
I suggest adding a test to check that the default value can be returned.

Reply:
Thanks @dnkolegov ! However I encounter this problem when I try to call `getCommitSha()` from unit test.

```
Encountered 1 failing test in test/integration/GatewayDiamond.t.sol:GatewayActorDiamondTest
[FAIL. Reason: FunctionNotFound(0x444ead5100000000000000000000000000000000000000000000000000000000); counterexample: calldata=0xcfe9aa240000000000000000000000000000000000000000000000000000000000000522 args=[1314]] testGatewayDiamond_Deployment_Works_Root(uint64) (runs: 0, μ: 0, ~: 0)
```

It can be easily reproduced by adding just one line of code in UT
```
require(gatewayDiamond.getter().getCommitSha() == 0, "unexpected totalSubnets");
```
This is weird. I wonder any step is missing here? Have you seen the same error before?

Reply:
Yes, because you need to run `make build-selector-library` if you add, remove or change Diamond functions

Reply:
> Yes, because you need to run `make build-selector-library` if you add, remove or change Diamond functions

It worked perfectly! Thanks a lot for the help!